diff --git a/node_modules/hexo-renderer-markdown-it/node_modules/markdown-it/lib/index.js b/node_modules/hexo-renderer-markdown-it/node_modules/markdown-it/lib/index.js
index afec8d8..d3ba7fb 100644
--- a/node_modules/hexo-renderer-markdown-it/node_modules/markdown-it/lib/index.js
+++ b/node_modules/hexo-renderer-markdown-it/node_modules/markdown-it/lib/index.js
@@ -11,7 +11,7 @@ var ParserBlock  = require('./parser_block');
 var ParserInline = require('./parser_inline');
 var LinkifyIt    = require('linkify-it');
 var mdurl        = require('mdurl');
-var punycode     = require('punycode');
+var punycode     = require('punycode/');
 
 
 var config = {
